addons/mail/models/mail_link_preview.py

```diff
@@ -142,3 +147,16 @@ def _gc_mail_link_preview(self):
             ('message_id', '=', False),
             ('create_date', '<', fields.Datetime.now() - timedelta(days=lifetime)),
         ], order='create_date ASC', limit=1000).unlink()
+
+    @api.model
+    def _init_ignore_list(self):
+        try:
+            base_url = urlparse(self.get_base_url())
+            port = base_url.port or False
+        except ValueError:
+            return
+        base_ignore_pattern = f".*{base_url.hostname}"
```

Comment:
Does it actually need to start with `.*`?
Especially if it doesn't end with it, it looks inconsistent.

Reply:
Look like it's not required indeed

Reply:
Turns out, yes, you do need it